lib/currency_cloud/errors/error_utils.rb

```diff
@@ -10,5 +10,18 @@ def platform
                        end
       "#{base}#{implementation}"
     end
+
+    REDACTED_PARAMS = [:api_key, :login_id, :token].freeze
+
+    def redacted_params(params)
+      redacted = params.dup
```

Comment:
I'm in-place updating the `redacted` instance, wanted to ensure I don't update the actual `params`, not sure where those are used. if not duping, this should be `redact_params!` instead